cmd/kops/update_cluster.go

```diff
@@ -206,6 +200,34 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 		}
 	}
 
+	lifecycleOverrideMap := make(map[cloudup.Phase]fi.Lifecycle)
+
+	for _, override := range c.PhaseOverrides {
+
+		values := strings.Split(override, "=")
+		phaseName := values[0]
+		lifecycle := values[1]
+
+		phaseOverride, err := getPhase(phaseName)
+		if err != nil {
+			return fmt.Errorf("incorrect phase name in overides: %v", err)
```

Comment:
Include the phaseName in the log message (or the `override` value)

Reply:
Done in the getPhase func.